README.md

```diff
@@ -0,0 +1,383 @@
+        COUNT(*) as download_count,
+    FROM `bigquery-public-data.pypi.file_downloads`
+    WHERE timestamp BETWEEN TIMESTAMP_ADD(CURRENT_TIMESTAMP(), INTERVAL -366 DAY) AND TIMESTAMP_ADD(CURRENT_TIMESTAMP(), INTERVAL -1 DAY)
+        AND details.installer.name = "pip"
+    GROUP BY
+        project
+    ORDER BY
+        download_count DESC
+    LIMIT 100
+```
+
+Downloads for a given version
+-----------------------------------------
+
+pypinfo supports `PEP 440 version matching <https://peps.python.org/pep-0440/#version-matching>`_.
+
+We can use it to query stats on a given major version.
+
+```console
+
+    $ pypinfo -pc 'pip==21.*' pyversion version
+    Served from cache: False
+    Data processed: 34.45 MiB
+    Data billed: 35.00 MiB
+    Estimated cost: $0.01
+
+    | python_version | version | percent | download_count |
+    | -------------- | ------- | ------- | -------------- |
+    | 3.6            | 21.3.1  |  78.74% |         10,430 |
+    | 3.8            | 21.3.1  |   7.81% |          1,034 |
+    | 3.7            | 21.2.1  |   3.59% |            476 |
+    | 3.7            | 21.3.1  |   2.60% |            345 |
+    | 3.7            | 21.0.1  |   2.25% |            298 |
+    | 3.8            | 21.0.1  |   1.58% |            209 |
+    | 3.8            | 21.2.1  |   1.42% |            188 |
+    | 3.7            | 21.1.2  |   0.81% |            107 |
+    | 3.9            | 21.3.1  |   0.69% |             92 |
+    | 3.8            | 21.1.1  |   0.51% |             67 |
+    | Total          |         |         |         13,246 |
+```
+
+We can also use it to query stats on an exact version:
+
+```console
+
+    $ pypinfo -pc 'numpy==1.23rc3' pyversion version
+    Served from cache: False
+    Data processed: 34.01 MiB
+    Data billed: 35.00 MiB
+    Estimated cost: $0.01
+
+    | python_version | version   | percent | download_count |
+    | -------------- | --------- | ------- | -------------- |
+    | 3.9            | 1.23.0rc3 |  63.33% |             38 |
+    | 3.8            | 1.23.0rc3 |  28.33% |             17 |
+    | 3.10           | 1.23.0rc3 |   8.33% |              5 |
+    | Total          |           |         |             60 |
+```
+
+</details>
+
+
+Installation
+------------
+
+pypinfo is distributed on **PyPI** as a universal wheel and is available on
+Linux/macOS and Windows and supports Python 3.7+.
+
+This is relatively painless, I swear.
+
+**Create project**
+
+1. Go to [https://bigquery.cloud.google.com](https://bigquery.cloud.google.com).
+2. Sign up if you haven't already. The first TB of queried data each month
+   is free. Each additional TB is $5.
+
+3. Go to [https://console.developers.google.com/cloud-resource-manager](https://console.developers.google.com/cloud-resource-manager) and click CREATE PROJECT if you don't already have one:
+    
+   ![create](https://user-images.githubusercontent.com/1324225/47172949-6f4ea880-d315-11e8-8587-8b8117efeae9.png "CREATE PROJECT")
```

Comment:
Thanks for including alt texts here, I think we can improve them a bit or leave them out?